viewer/src/glsl/base/updateFragmentColor.glsl

```diff
@@ -2,24 +2,50 @@
 #pragma glslify: hsv2rgb = require('../color/hsv2rgb.glsl')
 #pragma glslify: packIntToColor = require('../color/packIntToColor.glsl')
 
-#ifdef COGNITE_COLOR_BY_TREE_INDEX
+const int RenderTypeColor = 1;
+const int RenderTypeNormal = 2;
+const int RenderTypeTreeIndex = 3;
+const int RenderTypePackColorAndNormal = 4;
 
-void updateFragmentColor(vec3 color, float treeIndex, vec3 normal) {
-    color = packIntToColor(treeIndex);
-    float amplitude = max(0.0, dot(normal, vec3(0.0, 0.0, 1.0)));
-    gl_FragColor = vec4(color * (0.4 + 0.6 * amplitude), 1.0);
-}
+uniform int renderType;
```

Comment:
Should this uniform (and probably 'enum') be externalized and provided as an argument to upgradeFragmentColor?